lib/absinthe/phase/subscription/result.ex

```diff
@@ -5,10 +5,54 @@ defmodule Absinthe.Phase.Subscription.Result do
   # subscription
 
   alias Absinthe.Blueprint
+  alias Absinthe.Blueprint.Continuation
+  alias Absinthe.Phase
 
   @spec run(any, Keyword.t()) :: {:ok, Blueprint.t()}
-  def run(blueprint, topic: topic) do
+  def run(blueprint, options) do
+    topic = Keyword.get(options, :topic)
+    prime = Keyword.get(options, :prime)
```

Comment:
Thoughts on a more descriptive name like `after_connect` or `initial_response` or `initial` or something similar?

Reply:
FWIW, I'm _fine_ with prime, just there may be something more apt/intuitive here.

Reply:
`prime` was suggested by @benwilson512 and I kind of like it since it's both terse and descriptive. `after_connect` doesn't really fit because the "connection" has nothing to do with this. It's after a successful *subscription* that it's sent. The other two options I don't really have any problem with but nor do I think they really have anything to recommend them over `prime`.

Reply:
Sure. `after_subscribe` also viable. But either way, if it LGTBen, then it LGTM. LOL.